core/src/hash_table/entry.rs

```diff
@@ -67,7 +67,7 @@ impl Entry {
 
     /// returns true if the entry is valid
     pub fn validate(&self) -> bool {
-        // always valid iff immutable and new() enforces validity
+        // always valid if immutable and new() enforces validity
```

Comment:
I stumbled upon this as well. `iff` stands for `if and only if`

Reply:
I'll update the PR with a rephrased version then.

Reply:
iff is totally a word, haha